### PR TITLE
By default only do captcha  in  cf bouncer

### PR DESCRIPTION
**Pull request by sbs2001**

### CODE REVIEW

File: crowdsec-docs/docs/bouncers/cloudflare-workers.mdx [hunk 1]
```diff
@@ -97,6 +97,14 @@ You can run `sudo crowdsec-cloudflare-worker-bouncer -g <CLOUDFLARE_TOKEN1>,<CLO
 
 :::
 
+:::note
+By default the generated config only sets up captcha remediation for all the zones.
```
Comment (rr404) [inline crowdsec-docs/docs/bouncers/cloudflare-workers.mdx:101]: we should mention the word "challenge" right? that's the vocabulary they use in CF right?

File: crowdsec-docs/docs/bouncers/cloudflare-workers.mdx [hunk 2]
```diff
@@ -97,6 +97,14 @@ You can run `sudo crowdsec-cloudflare-worker-bouncer -g <CLOUDFLARE_TOKEN1>,<CLO
...
+You can change this by editing `cloudflare_config.accounts[*].zones[*].actions` and `cloudflare_config.accounts[*].zones[*].default_action` in the config to support ban action. Or make ban action the default. 
+
+To support ban and captcha remediations simultaneously, you can set `cloudflare_config.accounts[*].zones[*].actions` to `["captcha", "ban"]`
+IP's originating from decisions with `captcha` action would be handled by captcha remediation and decisions with `ban` action would be handled by ban remediation. Decisions with other actions would be applied remedications specified by `cloudflare_config.accounts[*].zones[*].default_action` value.
```
Comment (rr404) [inline crowdsec-docs/docs/bouncers/cloudflare-workers.mdx:105]: can you put the "Decisions with .. in a new line ?"

**Review by rr404 — CHANGES_REQUESTED** [review]

File: crowdsec-docs/docs/bouncers/cloudflare-workers.mdx (hunk 1 above)
Comment (sbs2001) [reply crowdsec-docs/docs/bouncers/cloudflare-workers.mdx]: fixed

File: crowdsec-docs/docs/bouncers/cloudflare-workers.mdx (hunk 2 above)
Comment (sbs2001) [reply crowdsec-docs/docs/bouncers/cloudflare-workers.mdx]: fixed

**Comment by LaurenceJJones** [comment]
Please merge main as docusarus v3 has been integrated

**Pull request closed without merging**